src/core_plugins/tagcloud/public/tag_cloud_vis.js

```diff
@@ -13,6 +13,7 @@ visTypes.register(function TagCloudProvider(Private) {
   return new TemplateVisType({
     name: 'tagcloud',
     title: 'Tag cloud',
+    implementsRenderComplete: true,
```

Comment:
required for Reporting to listen check the render-counter property, instead of using a timeout